src/dotnet/APIView/APIViewUnitTests/CodeFileHelpersTest.cs

```diff
@@ -52,7 +52,7 @@ public CodeFileHelpersTests(ITestOutputHelper output)
         [Fact]
         public void ComputeAPITreeDiff_Generates_Accurate_TreeDiff()
         {
-            var expectedResult =  new List<(string id, string diffKind)>
+            /*var expectedResult =  new List<(string id, string diffKind)>
```

Comment:
Test will be added back to verify against actual package.

Reply:
I have added one test case back. I will add remaining test cases in a separate PR